src/main/java/edu/kis/vh/nursery/newStackImplementaion/StackInterface.java

```diff
@@ -0,0 +1,15 @@
+package edu.kis.vh.nursery.newStackImplementaion;
+
+public interface StackInterface {

```

Comment:
Names of operations do not correspond to operating the stack, thus do not reflect the interface name.